internal/diffmapper/diffmapper.go

```
@@ -70,14 +70,8 @@ func GetClosestPrOffset(prLineNr int) (int, error) {
 		return 0, fmt.Errorf("you must run GetMap before getting the closest pr offset")
```

Error handling is missing for strconv.Atoi(hunkStart[0]). If the conversion fails, the error is ignored, which could lead to unexpected behavior.
 The code assumes hunkStart will always have at least one element after splitting, but doesn't validate this, which could lead to an index out of range 
panic.